beacon_node/network/src/service.rs

```diff
@@ -824,23 +813,17 @@ impl<T: BeaconChainTypes> NetworkService<T> {
 
     /// Handle a message sent to the network service.
     async fn on_validator_subscription_msg(&mut self, msg: ValidatorSubscriptionMessage) {
-        match msg {
+        if let Err(e) = match msg {
```

Comment:
See earlier comments - i dont see errors returned from the `validator_subscriptions`. 

If we do return an error, it might be worth also logging the subscription message type